news/2018-07-16-cwc-2018-recap-grand-finals.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "osu!catch World Cup 2018: Grand Finals Recap"
+date: 2018-07-16 09:00:00 +0000
+---
+
+Of our three fruit-obsessed finalists, only one team could be crowned this year's osu!catch World Cup Champions. Chile, China and the United States – through the fire and the falling, which would reign supreme? All the juicy details can be found within!
+
+![](/wiki/shared/news/banners/CWC_2018_banner.jpg)
+
+Greetings once again everyone, [JBHyperion](https://osu.ppy.sh/users/4879508) here for your final osu!catch World Cup 2018 update. The sun has set on the year's premiere fruit-catching tournament at last, and a team of worthy Champions have been crowned.
+
+It was truly a tournament for the ages, as old Gods fell, and new warriors arose to forever etch their names in history. There were ties and tiebreakers aplenty. Some crept by on the narrowest of margins, whilst others, sadly, fell agonizingly short. Needless to say, my expectations have been exceeded on almost every conceivable level!
+
+There's no need to simply take my word for it though – naturally you can find videos of both Finals Week 2 matches (and previous' weeks if you need a refresher) on the [osu!live Twitch channel](https://www.twitch.tv/osulive). If you can spare some time, you absolutely will not regret watching all the action as it went down!
+
+<iframe src="https://player.twitch.tv/?autoplay=false&video=v284639720" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="100%"></iframe>
```

Comment:
just to verify, you have copied this directly from a twitch embed on their site and not just rehashed the format from an old article right?

Reply:
If it worked in previous articles and I change the ref after "video=", would it not work here? I checked the embed code again and it's the exact same after I remove the "a ref" etc. parts

Reply:
should be fine then, let's hope 🙏